src/app/content/highlights/components/SummaryPopup/__snapshots__/ChapterFilter.spec.tsx.snap

```diff
@@ -252,8 +251,7 @@ input:checked + .c10 .c11 {
 
 @media screen and (max-width:75em) {
   .c0.c0.c0 {
-    left: -1.6rem;
-    max-width: calc(100vw - 1.6rem);
+    max-width: calc(100vw - 4.800000000000001rem);
```

Comment:
please round this or find some other way to make the floating point math not produce a crazy number